bridge/server/models/webhook-config-yaml.ts

```diff
@@ -110,6 +113,7 @@ export class WebhookConfigYaml implements WebhookConfigYamlResult {
         const parsedConfig = this.parseConfig(curl);
         parsedConfig.secrets = webhook.envFrom;
         parsedConfig.sendFinished = webhook.sendFinished ?? false;
```

Comment:
Shouldn't be `sendFinished` also be true? Because if I create a new subscription and select triggered, sendFinished is also set to `true` as default

Reply:
The behaviour of the `webhook-service` is as follows:
- if the `sendStarted` flag is not set in the `webhook.yaml` it sends the .started event by default
- if the `sendFinished` flag is not set in the `webhhook.yaml` it doesn't send the .finished event by default

Parsing the file, will behave the same, but the default when creating a new webhook subscription is true for both, `sendStarted` and `sendFinished`.